src/index.html

```diff
@@ -22,7 +22,7 @@
   DEALINGS IN THE SOFTWARE.
 -->
 
-<html>
+<html spellcheck="false">
```

Comment:
I think @RaymondLim was suggesting to use that technique to apply `spellcheck="false"` to the `html` tag so it's not hard-coded in index.html. Probably only need to apply it to the `body` tag.